web/src/settings_account.js

```diff
@@ -731,6 +732,7 @@ export function set_up() {
                 form_id: "change_email_form",
                 on_click: do_change_email,
                 on_shown() {
+                    ui_util.place_caret_at_end($("#change_email_form input"));
                     $("#change_email_form input").trigger("focus");
```

Comment:
Can we remove this focus code, since `place_caret_at_end` already focusses the element.

Reply:
oh sorry, I forgot to remove it. pushing the updated PR in 5 mins.